internal/store/store.go

```diff
@@ -390,6 +420,17 @@ func (bs *BlockStore) PruneBlocks(height int64, state sm.State) (uint64, int64,
 	if err != nil {
 		return 0, -1, err
 	}
+	bs.blocksDeleted += int64(pruned)
+
+	if bs.compact && bs.blocksDeleted >= bs.compactionInterval {
+		// Error on compaction should not be reason to halt the chain
+		// TODO add logger to keep track of error
+		err = bs.db.Compact(nil, nil)
+		if err != nil && bs.logger != nil {
```

Comment:
same comment about logger

Reply:
What Andy pointed out is that these errors won't cause a panic in the end. I was tricked by the way this is done in 0.37 where it would panic and I don't think a compaction failure should be cause for a panic. I think we should remove the loggers alltogether given this?

Reply:
> I think we should remove the loggers alltogether given this?

Agree